src/Transits.jl

```diff
@@ -81,8 +81,7 @@ julia> ld(orbit, 0u"d", 0.1)
 function compute(ld::AbstractLimbDark, orbit::AbstractOrbit, t, r)
     coords = Orbits.relative_position(orbit, t)
     z = sqrt(coords[1]^2 + coords[2]^2)
-    b = z / orbit.r_star
```

Comment:
why this change? As far as I understand the `b` used for the `compute(ld, b, r)` is unitless, so we need this conversion when calculating the z coordinate from the orbit. I could be wrong though!

Reply:
Right, my other change [here](https://github.com/JuliaAstro/Transits.jl/pull/6/files#diff-b40b64af6306e4b5cd8e15b06aaf3647ed6c0dfcd28950ee42823ac15aa0eabfR131) keeps things unitless further upstream =]

Reply:
So I think it would be good to do the same thing in `simple.jl`

Reply:
Okay, that seems reasonable, I'll have to check the `flip` method still works, if so I'll throw that into a separate PR.